test/integration/targets/vmware_vmotion/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+# Test code for the vmware_vmotion module.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_attach_host: true
+    setup_virtualmachines: True
```

Comment:
Remove the `setup_virtualmaches: True`, or `prepare_vmware_tests` will create VM for you. I don't think you need that and it slow down the test.

Reply:
Done